tests/unittest/disaggregated/test_openai_disagg_service.py

```
@@ -752,3 +752,56 @@ def test_deserialize_invalid_type_raises(self):
+
+    async def _capture_send(req, *args, **kwargs):
+        sent["ctx"] = req
+        raise RuntimeError("stop after ctx dispatch")
+
+    service._ctx_client.send_request = AsyncMock(side_effect=_capture_send)
+
+    request = CompletionRequest(model="model", prompt=[1] * 8, stream=False)
+    try:
+        await service._send_disagg_request(request, None)
+    except Exception:
+        pass  # we only care about which object reached the router vs the client
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)